akowalska622/growme-rn
# Feature/grow 60 text element implementation

## Context

I implemented `Text` component :D 
It receives two props:
* `variant`: `"titleOne" | "titleTwo" | "titleThree" | "headline" | "body" | "callout" | "subhead" | "footnote" | "caption"` with default `body`,
* `color`: `"disabled" | "placeholder" | "primary" | "background" | "card" | "text" | "border" | "notification"` with default `text`.

The color palette (which is appropriate for any appearance, `dark` or `light`) in styles is used to determine the color of the text.

I also wrote a story for the `Text` component.

## Resources

* [Figma mocks](https://www.figma.com/file/TXTQ7499AZmlMM43JG7oLe/GrowMe?node-id=121%3A50)

## Test path

1. Run `yarn install` to install project dependencies.
2. Run `yarn storybook` in the first terminal.
3. Run `yarn start` in the second terminal.
3. Open the app and go to the Storybook tab.
4. Find the `Text` component and play around with it.

⚠️ Unfortunately, for some reason the knobs ("customization" of the component) don't work on the device. You should customize the component through the Storybook web server.

**Expected result**:

Everything is as in the mocks.

## Checklist

- [ ] I checked the diff
- [ ] I formatted the code
- [ ] I wrote tests
